lutcha.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# vim:set ts=8 sts=8 sw=8 tw=80 noet:
+
+from module import Module, MUC
+from urllib.parse import quote as urlencode
+
+class Lutcha(Module):
+	def __init__(self, **keywords):
+		super(Lutcha, self).__init__([MUC], name=__name__, **keywords)
+
+	def muc_msg(self, msg, **keywords):
+		parts = msg.split(' ', 1)
+		if len(parts) == 2 and parts[0][0] == "!" and parts[0][1:].lower() in ["bash", "doc", "lutcha"]:
+			result = self.handle(msg)
```

Comment:
Besser wäre es, hier nicht die gesamte `msg` zu übergeben, sondern `parts` (bzw `parts[0][1:]` als Befehl und `parts[1].strip()` als Daten). Dann kann in `handle()` der Fall, dass mehrere aufeinanderfolgende Leerzeichen vorkommen, einfach behandelt werden.